### ISSUE TITLE
NTR: Prenatal ultrasound: lungs enlarged (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9008]
New term request Prenatal ultrasound: lungs enlarged (elthat_160820041254):

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide the disease context for this term?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/219000 "Markedly enlarged echogenic lungs were demonstrable at 16 and 17 weeks of gestation."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000613
- label: Enlarged fetal lungs
- parents: [Abnormal fetal pulmonary morphology (HP:0034243)](https://hpo.jax.org/app/browse/term/HP:0034243)
- pmid: PMID:7747754
- def: The size (volume) of the lungs of a fetus above the upper limit of normal for the gestational age.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9008
- synonyms:
